main.go

```diff
@@ -20,6 +21,13 @@ func main() {
 	// Initialize global randomness
 	rand.Seed(time.Now().UnixNano())
 
+	// Initialize watchman
+	err := watchman.Configure("192.168.122.1", "9125", "docker.metrics")
```

Comment:
These values can't be hard-coded. They need to be passed into the runtime with CLI arguments.